README.md

```diff
@@ -109,6 +109,8 @@ To work around the issue, edit your `jvm.config` settings by adding the followin
 
 * If the installation of Presto fails with `Python script has been killed due to timeout after waiting 1800 secs`, then the `wget` for either the Presto RPM or `presto-cli` JAR has timed out. To increase the timeout, increase the `agent.package.install.task.timeout` setting in `/etc/ambari-server/conf/ambari.properties` on the Ambari server host. Make sure to restart the Ambari server for the change to take effect. To resume, either hit the Retry button in the installation wizard, or finish the wizard and then install all Presto components individually by navigating to the relevant host and selecting Re-install. The components can be installed manually in any order but when starting the components, make sure to start the Presto coordinator last. If the installation keeps timing out, we suggest downloading the RPM and JAR outside the installation process, uploading them somewhere on your network and editing `package/scripts/download.ini` with the new URLs.
 
+* At the moment, upgrading Presto from Ambari is not possible. Even though Ambari provides the capability to upgrade software, we didn't get a chance to implement the integration. If many users request this feature, we'll probably add it to the next release.
```

Comment:
I think you should provide the steps on how to upgrade. Which is to simply explain they need to uninstall and reinstall the new rpm. Then create a github issue on this repo for upgrade.